assets/js/common/ProfileMenu/ProfileMenu.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import Button from '@common/Button';
+
+import { EOS_ACCOUNT_CIRCLE_OUTLINED } from 'eos-icons-react';
+
+
+
+function ProfileMenu({username, email, logout}) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div className="relative inline-block text-left z-10">
+      <div>
+        <Button
+          type="transparent"
+          onClick={toggleMenu}
+          className="flex items-center"
+        >
+          <EOS_ACCOUNT_CIRCLE_OUTLINED
+            size="24"
+            color="currentColor"
+            className="text-gray-600 hover:text-gray-300"
+          />
+          <span className="text-gray-600 ml-2">{username}</span>
+          <svg
+            className="ml-2 h-5 w-5"
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 20 20"
+            fill="currentColor"
+            aria-hidden="true"
+          >
+            <path
+              fillRule="evenodd"
+              d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
+              clipRule="evenodd"
+            />
+          </svg>
+        </Button>
+      </div>
+
+      {isOpen && (
+        <div
+          className="origin-top-right absolute right-0 mt-2 w-56 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 focus:outline-none"
+          role="menu"
+          aria-orientation="vertical"
+          aria-labelledby="menu-button"
+          tabIndex="-1"
+        >
+          <div className="py-1" role="none">
+            <span
+              role="menuitem"
```

Comment:
Hey man i think we could use the [headless ui libary](https://headlessui.com/react/menu) for the drop down.
Check out the Select.jsx component, as we already have it in the project. 
What do you think ?

Reply:
taking a look :+1:

Reply:
using it now :-)